### DISCUSSION TITLE
How i do a custom event (hscript(1.7.0(no source)))

### DISCUSSION THREAD

**Discussion by jasonroni** [YoshiCrafter29/YoshiCrafterEngine#102] [Q&A]
**Hscript cuz this is not psych LFMAO**
but really help

**Comment by ErenKaya31** [comment]
did the 1.7.0 out?

**Reply by jasonroni** [reply]
https://github.com/YoshiCrafter29/YoshiEngine/actions/runs/1995235930 official

**Comment by ErenKaya31** [comment]
@Jsa010 can you post a video to psych engine discussions pls https://user-images.githubusercontent.com/95748211/158996682-cc32e6f4-6dbf-4868-aa2e-25c953fc7964.mp4

**Reply by jasonroni** [reply]
no lolol
